pandas/core/arrays/integer.py

```diff
@@ -446,6 +450,7 @@ def astype(self, dtype, copy: bool = True) -> ArrayLike:
             casting
         """
         from pandas.core.arrays.boolean import BooleanDtype
+        from pandas.core.arrays.floating import _FloatingDtype
```

Comment:
why is the dtype _FloatingDtype? and not FloatingDtype?

Reply:
yep that sounds good

Reply:
Renamed